cleanup_utils/playbooks/load_balancer_cleanup.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Playbook to remove the stale load balancer fragments
+
+- hosts: all
+  become: True
+  vars:
+    DAYS_OLDER_THAN: 3
+    FRAGMENT_DIR: /etc/haproxy/conf.d
+    FRAGMENT_PATTERN: "integration*"
+    REMOVE_FRAGMENTS: false
+  tasks:
+    - name: Find load balancer fragment files older than "{{ DAYS_OLDER_THAN }}" days
+      find:
+        paths: "{{ FRAGMENT_DIR }}"
+        age: "{{ DAYS_OLDER_THAN }}d"
+        patterns: FRAGMENT_PATTERN
```

Comment:
This didn't work still for me, not sure why.

```bash
/etc/haproxy/conf.d# ls -alh
total 80K
drwx------ 2 root root  24K Feb 27 11:03 .
drwxr-xr-x 6 root root 4.0K Feb 27 05:43 ..
-rw-r--r-- 1 root root 2.5K Feb 27 10:43 00-main
-rw-r--r-- 1 root root  963 Feb 27 10:43 01-maintenance
-rw-r--r-- 1 root root  432 Feb 27 11:04 integration-173c93df
-rw-r--r-- 1 root root   86 Feb 27 07:59 integration-2fb77304
-rw-r--r-- 1 root root   86 Feb 17 10:57 integration-565ca74e
-rw-r--r-- 1 root root   86 Feb 17 10:57 integration-5e71da6d
-rw-r--r-- 1 root root   86 Feb 27 10:02 integration-e69225c6
-rw-r--r-- 1 root root  14K Feb 27 10:43 stage-3b4620f2
-rw-r--r-- 1 root root   86 Feb 27 10:43 stage-ce2c673c
```

The feb 17 ones (manually modified) didn't get deleted.

Reply:
@UmanShahzad, sorry about that. Let me check this once more and debug the issue.

Reply:
@UmanShahzad, it was gotcha where I though that ansible will use the value of the `FRAGMENT_PATTERN` variable doesn't, but it doesn't. Fixed it now.

Reply:
:+1: